Explaining the need for operator group during install docs (#27)
operator-framework/olm-docs: content/en/docs/Tasks/install-operator-with-olm.md

  [Once you've made your operator available in a catalog](/docs/tasks/make-operator-part-of-catalog/), or you've chosen an operator from the [list of operators available to be installed in the cluster](/docs/tasks/list-operators-available-to-install/), you can install your operator by creating a [`Subscription`](/docs/concepts/customresourcedefinitions/subscription/) to a specific [channel](/docs/concepts/glossary/#channel). 
---

> Before installing into a namespace, you will need to create an [operator group](/docs/concepts/crds/operatorgroup) in that namespace. The operator group must provide the namespaces that match the operator that you trying to install `installModes`.

Reviewer: ```suggestion
Before installing an Operator in a namespace, you will need to create an [OperatorGroup](/docs/concepts/crds/operatorgroup) in that namespace. The operator group must provide the namespaces that match the operator that you trying to install `installModes`.
```

Reviewer: `The operator group must provide the namespaces that match the operator that you trying to install `installModes`.` 

This sentence is confusing to me. Maybe it'll be clear if you elaborated on the part about "match"-ing the operator?

> For example installing an operator into `foo` namespace that will be cluster scoped you would create a operator group like:

Reviewer: ```suggestion
For example, to install a cluster scoped operator in  the namespace `foo`, you would create an OperatorGroup like:
```
